src/fastoad/gui/analysis_and_plots.py

```diff
@@ -445,7 +524,10 @@ def mass_breakdown_sun_plot(
         2,
     )
 
-    fig.update_layout(title_text="Mass Breakdown", title_x=0.5)
+    if not mission_name:
+        fig.update_layout(title_text="Mass Breakdown - Sizing Mission", title_x=0.5)
+    else:
+        fig.update_layout(title_text=f"Mass Breakdown - {mission_name}", title_x=0.5)
```

Comment:
Here also, repetition can be avoided.
```suggestion
    if not mission_name:
        mission_name = "Sizing Mission"
    fig.update_layout(title_text=f"Mass Breakdown - {mission_name}", title_x=0.5)
```
Note this is not just a matter of conciseness. If we ever want to modify these parts (title position, sunburst labels...), it will be far better if we have to do the change only once (avoids useless work, and risk of non consistency).